Utilities/ocvalidate/ValidatePlatformInfo.c

```diff
@@ -94,8 +94,8 @@ CheckPlatformInfoGeneric (
   }
 
   AsciiSystemUUID     = OC_BLOB_GET (&UserPlatformInfo->Generic.SystemUuid);
-  if (AsciiSystemUUID[0] != '\0' && !AsciiGuidIsLegal (AsciiSystemUUID)) {
-    DEBUG ((DEBUG_WARN, "PlatformInfo->Generic->SystemUUID is borked!\n"));
+  if (AsciiSystemUUID[0] != '\0' && AsciiStrCmp (AsciiSystemUUID, "OEM") != 0 && !AsciiGuidIsLegal (AsciiSystemUUID)) {
```

Comment:
This line is becoming sort of too long. In my opinion, split it into multiple lines. That is to say:
```
if (cond1
  && cond2
  && ...) {
  // statements
}
```
Please make sure that logical operators (e.g. `&&`) are indented by two spaces.

Also, I believe now we should also list this `OEM` check in [ocvalidate README](https://github.com/acidanthera/OpenCorePkg/blob/master/Utilities/ocvalidate/README.md), at `PlatformInfo->Generic` part.

Once done, I think it will be the time to merge this PR. Thanks for your contribution!